minetest/minetest
# Move build/android directory to root of project

Inspiration from https://github.com/minetest/minetest/pull/10797 but instead of rebase doing the `git mv build/android .` myself and fixing the relative paths used in android build as well

> The top-level `build/` folder is now in `.gitignore` as well.
> 
> This closes #9800.
> ## To do
> 
> This PR is Ready for Review.
> ## How to test
> 
> Now that `build/` is no longer used for VCS-committed Android files, building an installing Minetest can now be done using those 3 "standard" CMake commands:
> 
> ```
> cmake -Bbuild
> cmake --build build
> sudo cmake --build build --target install
> ```
